common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsOAuth2Configuration.java

```diff
@@ -53,4 +83,29 @@ private URL getEndpoint(URL root, String endpoint) {
 
     }
 
+    private OpenIdProviderConfiguration getOpenIdWellKnownConfigForAuthority() {
```

Comment:
Suggest we declare the return type `@Nullable` and javadoc that this method can return `null` if errors are encountered

Reply:
and for completeness, that callers should check the result before using it

Reply:
Addressed